tools/daily-builds/deb_daily_builds.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# This file is part of Checkbox.
+#
+# Copyright 2022-2023 Canonical Ltd.
+# Written by:
+#   Sylvain Pineau <sylvain.pineau@canonical.com>
+#
+# Checkbox is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+#
+# Checkbox is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Checkbox.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess
+
+from pathlib import Path
+
+
+def run(*args, **kwargs):
+    """wrapper for subprocess.run."""
+    try:
+        return subprocess.run(
+            *args, **kwargs,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as e:
+        print('{}\n{}'.format(e, e.output.decode()))
+        raise SystemExit(1)
+
+
+def main():
+    """Parse the checkbox monorepo to trigger deb daily builds in Launchpad."""
+    # First request code import (GitHub -> Launchpad)
+    run(
+       "./tools/release/lp-request-import.py "
+       "~checkbox-dev/checkbox/+git/checkbox",
+       shell=True, check=True)
+    projects = {}
+    for path, dirs, files in os.walk('.'):
+        if "debian" in dirs:
+            project_path = str(Path(*Path(path).parts))
```

Comment:
Not sure if this line is useful. Converting a string to a Path, then back to a string... but I might be missing something :)